source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,43 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - ``collMod``
+  - ``create``
+  - ``createIndexes``
+  - ``drop``
+  - ``dropDatabase``
+  - ``dropIndexes``
+  - ``refineCollectionShardKey``  
+  - ``rename``
+  - ``reshardCollection``
+  - ``shardCollection``
+
+  This includes operations that may create new collections such as 
+  :dbcommand:`mapReduce`, :pipeline:`$out`, and :pipeline:`$merge`. This also 
+  includes collections created implicitly from inserts. Only writes that produce 
+  CRUD events can occur during the migration.
+
+  .. note:: 
+   
+     Writes that produce DDL events on source collections outside of the 
+     :ref:`namespace filter <c2c-filtered-sync>` are allowed.
+
+- ``geoHaystack`` indexes are not supported.
+
+- :ref:`/reverse <c2c-api-reverse>` endpoint is not supported. You can't 
+  enable the ``reversible`` option in the :ref:`/start <c2c-api-start>` request.
+
+- You can't enable the ``enableUserWriteBlocking`` option in the ``/start`` 
+  request. 
+
+- You can't enable the ``createSupportingIndexes`` :ref:`sharding parameter 
+  <c2c-api-start-sharding>`. 
+
+- Indexes with inconsistent specs or that are missing on some shards are not 
+  supported. To check for index inconsistencies, see 
+  :ref:`manage-indexes-find-inconsistent-indexes`.
```

Comment:
@ajayvijayakumar123 is this a change from status quo mongosync GA? We have 'mongosync only syncs indexes that have consistent index specifications on all shards.' in the limitations today

Reply:
Yup. For 6.0+ migrations, mongosync _ignores_ inconsistent indexes and just doesn't build them on the destination. For pre-6.0 migrations, mongosync will error on these indexes.

Reply:
I'll make it clear that it'll error on pre-6.0 migrations

Reply:
thanks!